backend/src/App.ts

```diff
@@ -113,8 +113,9 @@ export class App {
         }
 
         if(!await isShopTableEmpty() && !await isAccountTableEmpty() && await isActiveTagTableEmpty()) {
-            console.log('...Bookmarks empty, inserting 2000 bookmarks...')
-            await this.insertRandomTestActiveTag(2000)
+            const activeTagInsertCount = 2000
+            console.log(`...Active Tags empty, inserting ${activeTagInsertCount} bookmarks...`)
```

Comment:
did you mean to leave this console.log?

Reply:
Yeah but it's not quite right, I need to change it to say it's inserting tags.

I do want these logs to print, it shows you what the container is doing when it spins up.

Reply:
I fixed it before merging